hasher-matcher-actioner/Makefile

```diff
@@ -0,0 +1,47 @@
+.PHONY: all docker upload_docker dev_create_configs dev_create_instance dev_destroy_instance dev_clear_configs clean
+
+shell-or-die = $\
+	$(eval sod_out := $(shell $(1); echo $$?))$\
+	$(if $(filter 0,$(lastword $(sod_out))),$\
+		$(wordlist 1, $(shell echo $$(($(words $(sod_out)) - 1))), $(sod_out)),$\
+		$(error $(1) exited with status $(lastword $(sod_out))))
+
+USER ?= $(call shell-or-die,whoami)
+REPOSITORY_NAME = hma-lambda-dev
+DOCKER_TAG ?= ${USER} # todo - change to ${USER}-latest?
```

Comment:
Makefiles aren't known for their help text but either somewhere here or in contributing.md could you explain how to configure this? (i.e. example command usage)